src/main/java/com/dev/cinema/Main.java

```diff
@@ -55,8 +57,13 @@ public static void main(String[] args) {
         ShoppingCartService shoppingCartService = (ShoppingCartService) injector
                 .getInstance(ShoppingCartService.class);
         shoppingCartService.addSession(movieSession, user);
-        System.out.println(shoppingCartService.getByUser(user));
+        ShoppingCart shoppingCart = shoppingCartService.getByUser(user);
+        System.out.println(shoppingCart.toString());
         shoppingCartService.clear(shoppingCartService.getByUser(user));
         System.out.println(shoppingCartService.getByUser(user));
+
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+        orderService.completeOrder(shoppingCart);
```

Comment:
let's call `addSession()` before completing order
and get cart from db
so that the shopping cart got from the db was really non-empty

Reply:
done, if I correctly understood you